vocabularies/Org.OData.Core.V1.xml

```diff
@@ -539,6 +539,17 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
         </Property>
       </ComplexType>
 
+      <Term Name="changed" Type="Edm.Boolean" Nullable="true">
+        <Annotation Term="Core.Description" String="Instance annotation on [added/changed entities in a delta message](https://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_AddedChangedEntity), `true` indicating that the entity existed and was changed when the delta message was produced" />
+        <Annotation Term="Core.LongDescription">
+          <String>The presence or absence of `@Core.changed` does not alter the idempotent nature of a delta message, nor does it alter the UPSERT nature of an added/changed entity.

```

Comment:
```suggestion
          <String>The presence or absence of `@Core.changed` does not alter the idempotency of a request containing a delta message, nor does it alter the UPSERT nature of an added/changed entity.
```
Not the message itself is idempotent (it could also appear in a response), but a request using the message is.

Reply:
The response message is also idempotent for the recipient, for example a mobile client that supports offline activity and pulls deltas from a service when online.

Reply:
See https://en.wikipedia.org/wiki/Idempotence: applying it twice (for example to the client's offline data cache) has the same effect as applying it once.

Reply:
Which was the motivating use case for adding deltas to OData V4. Using the same message format also in PATCH requests to collections was an afterthought many years later.